estatioapp/app/src/test/java/org/estatio/module/agreement/dom/Agreement_Test.java

```diff
@@ -584,7 +588,6 @@ public String getAtPath() {
                 endDate = new LocalDate(2023, 3, 30);
 
                 agreement = new AgreementForTesting();
-                agreement.setContainer(mockContainer);
             }
```

Comment:
If the test passes, then I guess that's ok, but I would imagine a need to inject some other service instead.